### ISSUE TITLE
Launching with optifine send me into vanilla MC

### ISSUE THREAD

**Issue by SyltonMC** [sp614x/optifine#3989]
I installed MC for 1.14.4 (To play in a server for that version) and the first time I installed it, it seemed to have work just fine as the shaders options was there, I quit playing for a day, come back ready to install shaders, and run MC through optifine like normal only for it to send me into MC without Optifine as the optifine info isn't on the bottom left of the menu, and shaders options are gone

Not sure how to reproduce, I've reinstalled MC, optifine and Java, with nothing working

## OptiFine Version
for 1.14.4 F5?

## Installation Method
Installed it through opening the .jar file with Java

**Comment by realjackasterisk** [comment]
you're sure you're launching the OptiFine profile in the launcher?

**Comment by SyltonMC** [comment]
> you're sure you're launching the OptiFine profile in the launcher?

yeah, I did it with the little pop up menu and from the installations options

**Comment by ayush06feb** [comment]
Clear all the previous installed versions and redo the installations. If that doesn't work, try manual installation.
